### DISCUSSION TITLE
Sensitivity

### DISCUSSION THREAD

**Discussion by Jonaleow04** [stm32duino/Arduino_Core_STM32#2212] [Q&A] [answered]
Anyone knows how to change the sensitivity of the sensor? What is the code for it. Help is much appreciated

**Comment by fpistm** [comment, marked as answer]
Hi @Jonaleow04 
How can we help you with this:

> Anyone knows how to change the sensitivity of the sensor? What is the code for it. Help is much appreciated

You don't tell us what you used? Which sensor? harware used (board, shield,...)? Software (Core version, ...)
